website/docs/routes/social-media.mdx

```diff
@@ -1702,7 +1702,13 @@ rule
 
 ### 热搜榜 {#wei-bo-re-sou-bang}
 
-<Route author="xyqfer" example="/weibo/search/hot" path="/weibo/search/hot" anticrawler="1" radar="1" rssbud="1"/>
+<Route author="xyqfer shinemoon" example="/weibo/search/hot/fulltext" path="/weibo/search/hot/:fulltext?" anticrawler="1" radar="1" rssbud="1"/>
+
+- 使用`/weibo/search/hot`可以获取热搜条目列表；
+- 使用`/weibo/search/hot/fulltext`可以进一步获取热搜条目下的摘要信息（不含图片视频）；
+    - 使用`/weibo/search/hot/fulltext?pic=true`可以获取图片缩略（但需要配合额外的手段，例如浏览器上的Header Editor等来修改referer参数为`https://weibo.com'，以规避微博的外链限制,否则图片无法显示。)
```

Comment:
There's is special utility in `lib/v2/weibo/utils.js` to handle this.
https://github.com/DIYgod/RSSHub/blob/6fb730b18ddbac07ae2b0362f46014b74f70eac7/lib/v2/weibo/search/hot.js#L34

Reply:
Thanks... Tony,

I tried, but seems the workaround doesn't help now (even though all expected **tvax** already replaced there) ?..like below:
```
请求 URL:
https://tvax1.sinaimg.cn/thumbnail/69002366gy1hk0ptrlj39j20m80xcag1.jpg
请求方法:
GET
状态代码:
403 Forbidden
远程地址:
----
引用者策略:
no-referrer
```